config/jobs/image-pushing/k8s-staging-sig-storage.sh

```diff
@@ -76,6 +91,7 @@ for repo in "${REPOS[@]}" "${BROKEN_REPOS[@]}"; do
       branches:
         # For publishing canary images.
         - ^master$
+        - ^main$
```

Comment:
Same logic that is implemented below is required here as well.

It looks good to me after this last change :tada: !

Thanks